WcaOnRails/app/views/competitions/mycomps.html.erb

```diff
@@ -0,0 +1,62 @@
+<% provide(:title, 'Upcoming competitions') %>
+
+<div class="container">
+  <h2><%= yield(:title) %></h2>
+
+  <table class="table">
```

Comment:
Yeah, it might be nice to have _nearby_competitions use `wca_table_for` as well, now that you bring it up =)